src/main/java/rx/exceptions/AssemblyStackTraceException.java

```diff
@@ -49,4 +42,48 @@ public AssemblyStackTraceException(String message) {
     public synchronized Throwable fillInStackTrace() { // NOPMD 
         return this;
     }
+    
+    /**
+     * Finds an empty cause slot and assigns itself to it.
+     * @param exception the exception to start from
+     */
+    public void attach(Throwable exception) {
```

Comment:
`attachTo`, otherwise it looks like you're attaching `exception` to `this` while in fact it's vice versa